designsystem/src/commonMain/kotlin/org/hisp/dhis/mobileui/designsystem/component/Legend.kt

```diff
@@ -0,0 +1,65 @@
+package org.hisp.dhis.mobileui.designsystem.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.HelpOutline
+import androidx.compose.material.ripple.LocalRippleTheme
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import org.hisp.dhis.mobileui.designsystem.theme.Ripple
+import org.hisp.dhis.mobileui.designsystem.theme.Spacing
+
+@Composable
+fun Legend(color: Color, text: String, onClick: () -> Unit) {
+    CompositionLocalProvider(LocalRippleTheme provides Ripple.CustomDHISRippleTheme) {
+        Column(modifier = Modifier.clickable(onClick = onClick)) {
+            Row(
+                modifier = Modifier
+                    .padding(Spacing.Spacing16, Spacing.Spacing8, Spacing.Spacing8, Spacing.Spacing6),
+                horizontalArrangement = Arrangement.Center,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Box(
+                    modifier = Modifier.size(Spacing.Spacing12)
+                        .clip(CircleShape)
+                        .background(color)
+                )
+                Text(
+                    text,
+                    Modifier.padding(start = Spacing.Spacing8)
+                        .weight(2f, true),
+                    style = MaterialTheme.typography.bodyMedium
+                )
+                Icon(
```

Comment:
Is this icon always the same or can it be custom?

Reply:
I'm not certain, Il'l have to ask Marcos, if needed we can make it custom